docusaurus/docusaurus.config.js

```diff
@@ -503,6 +503,37 @@ const config = {
           //   label: 'Blog',
           //   position: 'right'
           // },
+          {
+            type: 'dropdown',
+            label: 'Docs',
+            position: 'right',
+            items: [
+              {
+                label: 'Blog',
+                href: 'https://openziti.io',
```

Comment:
can these things get icons too?

Reply:
Yes, but with some stylesheet wizardry that's not obvious to me. They accept the same icon property as the existing social buttons, but the formatting is off.